src/sysupdate/sysupdate.c

```diff
@@ -1469,7 +1469,7 @@ static int verb_pending_or_reboot(int argc, char **argv, void *userdata) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "The --root=/--image= switches may not be combined with the '%s' operation.", argv[0]);
 
-        r = context_make_offline(&context, NULL);
+        r = context_make_offline(&context, NULL, true);
```

Comment:
as per current coding style we prefer if "opaque" parameters on function calls (i.e. those liek NULL, or true or false or literal numbers) are prefixed with a comment with the parameter name, i.e. please change this to:

```c
r = context_make_offline(&context, /* node= */ NULL, /* relax_no_transfer_check= */ true);
```

you get the idea.

Reply:
(here and everywhere)